charts/k8ssandra/values.yaml

```diff
@@ -134,15 +134,29 @@ backupRestore:
     # Set to true to use the same bucket across multiple clusters.
     multiTenant: false
 
-    # Accepted values are s3 and gcs
-    storage: s3
+    # Accepted values are s3, s3_compatible (S3 compatible storages) and gcs
+    storage: s3_compatible
```

Comment:
Should we set a default for the storage? I think we should have a check and fail the install if an invalid value is specified.